trick_source/java/makefile

```diff
@@ -5,6 +5,13 @@ include ${TRICK_HOME}/share/trick/makefiles/Makefile.common
 
 JAVAC ?= javac
 JAVAC_FLAGS = -g -Xlint:unchecked -Xlint:deprecation
+# delineate 'javac -version' for version number 
+JAVAC_VERSION := $(shell (${JAVAC} -version |& awk '{print $$2}') | awk -F ""."" '{print $$1}')
+
+# include java.se.ee for JAXB xml annotations (for trick-tv) in java version 9
+ifeq ($(JAVAC_VERSION),9)
```

Comment:
This should probably check for 9 or greater, or we'll be broken again in 10, right?

Reply:
Try this `javac -version | & awk '{print $2}' | & awk -F'[^0-9]' '{print $1'}`

Reply:
The "|&" pipe doesn't work on the mac.  The above expression also only returns "1" for all java versions 1.8 and below.  Here is my version using the correct utility, perl.
```
javac -version 2>&1 | perl -ne 'print /(\d+\.\d+)/'
```

Reply:
My version of the command above returned 2 digits, 9.0 or 1.8.  Here is a version that only returns one.
```
javac -version 2>&1 | perl -ne '/(\d+)\.(\d+)/ ; print +($1 == 1) ? $2 : $1'
```

Reply:
OK, @dbankieris tells me returning "1" for anything less than 9 is ok.  And not all versions follow the x.x version numbering.  So one more version that only gets the first digit(s)
```
javac -version 2>&1 | perl -ne 'print /(\d+)/'
```